Phenotypes.pm

```diff
@@ -112,7 +116,19 @@ sub new {
     my $version = $config->{db_version} || 'Bio::EnsEMBL::Registry'->software_version;
     my $assembly = $config->{assembly};
 
-    $DEFAULTS{file} = sprintf("%s_%s_%i_%s.bed.gz", $INC{$pkg}, $species, $version, $assembly);
+    my $dir = $self->params->[0];
```

Comment:
The input should be key value pairs - you can't assume the first input is a directory as command line users will put the phenotypes file here

Reply:
Thanks for checking. Indeed, that would be consistent. I've updated the parameter style in next PR136.